e2e/LongHaul/module/IotHub.cs

```diff
@@ -259,6 +261,57 @@ public async Task SetPropertiesAsync(string keyName, object properties, Logger l
             }
         }
 
+        public async Task InvokeDirectMethodOnLeafClientThroughEdgeAsync(string deviceId, string moduleId, string methodName, Logger logger, CancellationToken ct)
+        {
+            Debug.Assert(_moduleClient != null);
+            Debug.Assert(deviceId != null);
+            Debug.Assert(moduleId != null);
+
```

Comment:
Should customize the logger custom properties by adding what operation this logger is for.